assets/js/blocks/attribute-filter/style.scss

```diff
@@ -67,3 +67,10 @@
 		}
 	}
 }
+
+.editor-styles-wrapper  .wc-block-attribute-filter__button.wc-block-attribute-filter__button,
+.wc-block-attribute-filter__button.wc-block-attribute-filter__button {
+	padding: $gap-smaller $gap;
```

Comment:
What do you think about using the [`em()` mixin](https://github.com/woocommerce/woocommerce-blocks/blob/d8eba08ed53787caf00a071a08a4023b6c89b5cb/assets/css/abstracts/_mixins.scss#L275-L278) here so these values are in em instead of px (similar to how it's done [here](https://github.com/woocommerce/woocommerce-blocks/blob/bd24dd05a07f47a5c5abcbb11c3acf2cb9f8c025/assets/js/blocks/mini-cart/style.scss#L14))? That would make those sizes to adapt to the font size, so it should look better in different themes.